lib/runner.js

```diff
@@ -0,0 +1,43 @@
+const p = require('..')
+const TextLogStream = require('../lib/TextLogStream')
+
+const log = p.log.extend('runner')
+log.enabled = false
```

Comment:
That's the best I came up with. A runner logger will use namespace `barnard59:runner`, where the first segment is imported from the main module.

Disabled by default.